app/views/user_mailer/work_collaborator.html.slim

```diff
@@ -0,0 +1,9 @@
+p Hi #{@user.display_name} --
+
+p We just wanted to let you know that #{@work.owner.display_name} added you as a collaborator on #{@work.title}.
```

Comment:
Let's add the collection name here as well, since the work titles can be a bit obscure.